### ISSUE TITLE
Difference between OpenVR and SteamVR

### ISSUE THREAD

**Issue by Swoorup** [ValveSoftware/openvr#291]
What is the difference between OpenVR and SteamVR? Is OpenVR an interface and SteamVR an implementation?

**Comment by Ybalrid** [comment]
As I understand it : 

*SteamVR* is the runtime you install on your computer via Steam. *OpenVR* is the API that permit you to use various VR systems on your application via SteamVR, and as I understand, to code additional drivers for targeting other VR hardware.

SteamVR's main platform is obviously the Vive, but it ships with drivers giving it full compatibility with the Rift and Touch. Somebody wrote a driver to use the old Razer Hydras controllers as VR hand controllers. 

To sum things up : 

 - If you're an HMD or controller maker, you can make your system compatible with the OpenVR API by creating a driver for SteamVR. Every content using OpenVR will run on your hardware without the content developer doing additional work 😛 

 - If you're a content developer, using the OpenVR API you can target any platform where SteamVR runs. For example, every single Vive game out there will run on an Oculus Rift :wink:

**Comment by Swoorup** [comment]
Why won't they seperate steamVR and steam client? We would just like to use steamVR runtime without steam

**Comment by Ybalrid** [comment]
Same is true in the Oculus side of thing. You have to have Oculus Home running to use the Rift, even if you're not launching content from your Oculus Home Library.

**Comment by JohnathanBaird** [comment]
@Swoorup You can (now?) run SteamVR without having Steam. Steam is just a launcher/update service for SteamVR. If you have a Unity or Unreal game for example that is packaged to use SteamVR when you launch the game it can open SteamVR without ever opening Steam.

**Comment by JoeLudwig** [comment]
You are welcome to launch SteamVR without Steam and run content that way. Except for occasional bugs, this is a configuration we've supported since the Vive launch.

If you are in an enterprise where you want to distribute SteamVR internally without Steam on each machine, email vrlicensing@valvesoftware.com. We have an additional license for that.

OpenVR is the API. SteamVR is the implementation of that API.
